### PR TITLE
QDB-2329 - Simplify docker versionning.

### PR DESCRIPTION
**Pull request by vianneyPL**

### CODE REVIEW

File: qdb/README.md [hunk 1]
```diff
@@ -15,7 +27,9 @@ This repository contains the **Dockerfile** of [QuasarDB](http://www.quasardb.ne
 1. Install [Docker](https://www.docker.com/).
 1. Pull from docker: `docker pull bureau14/qdb`
 1. Alternatively: build an image with a [Dockerfile](https://hub.docker.com/r/bureau14/qdb/~/dockerfile/)
-1. Required files(replace {version} with the version you wish to use): [required-files]
+1. Required files(replace {version} with the version you wish to use):
```
Comment (mkurdej) [inline qdb/README.md]: Please fix small typos like this: `files(` <- missing space.

File: qdbsh/README.md [hunk 2]
```diff
@@ -14,6 +22,11 @@ This repository contains the **Dockerfile** of [QuasarDB](http://www.quasardb.ne
 
 1. Install [Docker](https://www.docker.com/).
 1. Pull from docker: `docker pull bureau14/qdbsh`
+1. Alternatively: build an image with a [Dockerfile](https://hub.docker.com/r/bureau14/qdbsh/~/dockerfile/)
+1. Required files(replace {version} with the version you wish to use):
```
Comment (mkurdej) [inline qdbsh/README.md]: Here too.

File: versions.sh [hunk 3]
```diff
@@ -41,6 +25,8 @@ function detect_version {
         return 1
     fi
 }
+<<<<<<< HEAD
```
Comment (mkurdej) [inline versions.sh]: Fix that!

File: versions.sh [hunk 4]
```diff
@@ -2,27 +2,11 @@
 
 source "utils.sh"
 
-QDB_VERSION=
-QDB_VERSION_PREFIX=
-QDB_CLEAN_VERSION=
-QDB_DEB_VERSION=1
+QDB_VERSION= # full version name, ex: 3.2.0master
```
Comment (mkurdej) [inline versions.sh]: You need to update `set_version.sh` script to take it into account.

Comment (vianneyPL) [reply versions.sh]: I don't think so, it's detected for the current archives being processed.

Comment (mkurdej) [reply versions.sh]: Good!

**Comment by mkurdej** [comment]
You have my blessing!

**Comment by solatis** [comment]
Where can I see the tags it's setting? It's focusing on the concept of "versions", while in fact it should focus on the concept of "tags". Docker itself is unaware of versions, and that's a good thing. It should only be considering a list of "tags", which are set using the "set_version.sh" script.

E.g. i would expect something like `TAGS=("nightly", "3.2.0", "3.2")` or `TAGS=("latest", "3.1.0", "3.1", "3")`. Not `QDB_LATEST_VERSION=`3.2.0`.

**Pull request closed without merging**